### ISSUE TITLE
[Feature Request] Downloading the State Snapshot with WGET

### ISSUE THREAD

**Issue by hadminh** [typesense/typesense#523]
Currently, there are 2 approaches, in order to replicate a Typsense Server.

Replicating SERVER_1 to SERVER_2
```
Approach 1)

retrieve collection GET ${SERVER_1}/collections/:collection
export collection GET ${SERVER_1}/collections/:collection/documents/export

create collection POST ${SERVER_2}/collections
import @documents.jsonl  ${SERVER_2}/collections/:collection/documents/import?action=create
```

```
Approach 2) 

create a snapshot POST ${SERVER_1}/operations/snapshot
mount the SERVER_1 volume into SERVER_2
copy state SNAPSHOT into SERVER_2
unmount SERVER_1 volume
start the Typesense-Server
```

The problem with the 2nd approach is that I have to create a ReadWriteMany volume so I can mount SERVER_1 to SERVER_2 and copy the snapshot state in. 

Can you provide a way so we can directly download the `state snapshot` into SERVER_2 using `wget`. I think this approach is more performant than Approach 1 (because of batch importing) and easier than Approach 2.

**Comment by kishorenc** [comment]
@hadminh 

With approach 2, can't you just `scp` the snapshot directory from `SERVER_1` to `SERVER_2`? This avoids the need to mount the volume.

**Comment by hadminh** [comment]
Yes, I believe `scp` is also a solution, but that also needs a setup :) `configmap`, private and public key setup, customizing the Typesense image. I'd rather not to do all that and use the default `Typesense` image. Feel free to close this or keep it as a low priority Feature Request.

Off topic: Please add `wget`, `vim`, `curl` to your Typesense image.
